app/templates/components/speaker-item.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="speaker-item">
+  {{#if speaker.image}}
+    <img alt="" class="ui image" src="{{speaker.image}}">
+  {{else}}
+    <img alt="" class="ui image" src="/images/placeholders/avatar.png">
```

Comment:
Put all of this in one statement
`<img alt="" class="ui image" src="{{if speaker.image speaker.image '/images/placeholders/avatar.png'}}">`

Also, add the speaker name as the `alt` attribute.